### PR TITLE
Plugin Does not support Godot 4.0 and higher update README

### PR DESCRIPTION
**Pull request by Dantali0n**
This really simple patch will go a long way for end users.

### CODE REVIEW

**Review by fanjin-z — APPROVED** [review]
Looks good! As discussed in #43, 4.0 won't be supported soon, this could clear up some confusion

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Debugger and utilities for working with Godot C# projects in VSCode.
 
 ## Requirements
 
-**Godot 3.2.2** or greater. Older versions of Godot are not supported.
+**Godot 3.2.2** till **4.0**. Older versions of Godot or **4.0** and higher are not supported.
```
Comment (raulsntos) [inline README.md]: How about using `>=` and `<` for the range? I feel like it's easier to understand.

```suggestion
**Godot  >= 3.2.2 and < 4.0**. Older versions of Godot or **4.0** and higher are not supported.
```

**Review by raulsntos — COMMENTED** [review]
Sorry, I totally missed this PR. It seems like a good compromise until we update the extension which may still take a while.

**Review by raulsntos — APPROVED** [review]
Looks good to me, thanks!

**Comment by Dantali0n** [comment]
Anything I can do to help this along? someone I could ping on email / irc etc?

**Comment by raulsntos** [comment]
Sorry, I messed up the rebase but the PR got merged in 7ab1157933039b7dc6cfba4f2f6265ac816e9f04

**Pull request closed without merging**